deskpro/deskpro-admin-frontend
# Feature/9142 data center page validation

Test steps:
- Navigation is not defined so change router 
   `'/tickets/locking': TicketLocking` to `'/tickets/locking': DataCenter` in /src/pages/index.ts
- Go to navigation `Features > Locking` for a test.
- Change first selectbox state to Select...

Expected behavior:
- Validation message "This field is required."
